### PR TITLE
Make the symmetric-enc example an example

### PR DESCRIPTION
**Pull request by greghaynes**
Making some minor tweaks to the doc example for symmetric encryption so
it is an actual, runable example.

### CODE REVIEW

**Comment by reaperhulk** [comment]
We create a key and iv in a `testsetup` block right now. We should remove that block if we want to make this doctest more clearly runnable.

**Comment by reaperhulk** [comment]
@greghaynes if you get a chance to update this PR with the fixes @alex and myself have requested we can merge it!

**Comment by greghaynes** [comment]
Yep, got a bit caught up with some holiday stuff but ill try and update soon.

Thanks!

**Comment by reaperhulk** [comment]
retest this please

File: docs/hazmat/primitives/symmetric-encryption.rst [hunk 1]
```diff
@@ -35,9 +29,12 @@ in an "encrypt-then-MAC" formulation as `described by Colin Percival`_.
 
     .. doctest::
 
+        >>> import os
         >>> from cryptography.hazmat.primitives.ciphers import Cipher, algorithms, modes
         >>> from cryptography.hazmat.backends import default_backend
         >>> backend = default_backend()
+        >>> key = os.urandom(32)
+        >>> iv = os.urandom(32)
```
Comment (reaperhulk) [inline docs/hazmat/primitives/symmetric-encryption.rst]: IV should actually be 16 (the block size of AES is 16 bytes). This should fail in the `validate_for_algorithm` check, but we won't know until Travis finishes up :)

**Pull request merged**